pkg/db/v1alpha3/common/helper.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+import "os"
+
+func GetEnvWithDefault(name, defaultValue string) string {
```

Comment:
https://github.com/kubeflow/katib/blob/master/pkg/controller.v1alpha3/consts/const.go#L37

Reply:
already implemented

Reply:
Gotcha, going to reuse it.